src/ApiClient/models/InternalComms/InternalError.ts

```diff
@@ -136,17 +137,20 @@ export default class InternalError<T extends ErrorCode = ErrorCode> {
             origin.config.headers["Authorization"] = "*********";
         }
 
-        let debuginfo = JSON.stringify({ addon, origin, config: configOptions, stack });
+        let debuginfo = JSON.stringify({ addon, origin, config: configOptions, stack }, null, 2);
+        debuginfo = debuginfo.replace(/\\n/g, String.fromCharCode(10)); // replace with a litteral newline
```

Comment:
```suggestion
        debuginfo = debuginfo.replace(/\\n/g, "\n"); // replace with a litteral newline
```

Reply:
doesnt work, the newline gets garbled on display (it's just `\\n`) instead of (an actual newline)

Reply:
![image](https://user-images.githubusercontent.com/25136265/151635232-0d53c5b5-24da-4659-969c-63edb118da85.png)
Unable to replicate? It's not impossible that its getting reescaped later down the line

Reply:
it gets unescaped on the (react) text